zeta-cpi/tests/zeta_cpi.ts

```diff
@@ -125,6 +127,25 @@ describe("zeta-cpi", () => {
     console.log("Your transaction signature", tx);
   });
 
+  it("Init spread account via CPI", async () => {
```

Comment:
Note: no tests for `position_movement` and `transfer_excess_spread_balance` because I'd need to set up a 2nd user to trade and therefore get into a position and it's just too much hassle IMO